tests/integration/targets/omevv_baseline_profile/tests/create_multiple_cluster.yaml

```diff
@@ -0,0 +1,120 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: "Please define the following variables: hostname, vcenter_username
+      and vcenter_password."
+  when:
+    hostname is undefined or vcenter_username is undefined or vcenter_password
+    is undefined
+
+- name: Set prerequisite
+  ansible.builtin.include_tasks: _helper_create.yml
+
+- block:
+    - name: Create a baseline profile with multiple
+        cluster (Check mode - Changes expected)
+      dellemc.openmanage.omevv_baseline_profile: &bp_multiple_cluster
+        state: present
+        name: OMAM_Baseline_Profile
+        repository_profile: OMAM_Firm_Repo
+        cluster:
+          - Raj-Cluster-1
```

Comment:
please don't hardcode cluster name, fetch is dynamically and then pass as required.